airflow/providers/google/cloud/hooks/vertex_ai/auto_ml.py

```diff
@@ -703,6 +705,10 @@ def create_auto_ml_forecasting_training_job(
         :param sync: Whether to execute this method synchronously. If False, this method will be executed in
             concurrent Future and any downstream object will be immediately returned and synced when the
             Future has completed.
+        :param window_stride_length: Optional. Step length used to generate input examples. Every 
+        ``window_stride_length`` rows will be used to generate a sliding window.
+        :param window_max_count: Optional. Number of rows that should be used to generate input examples. If the 
+        total row count is larger than this number, the input data will be randomly sampled to hit the count.
```

Comment:
```suggestion
        :param window_stride_length: Optional. Step length used to generate input examples. Every 
            ``window_stride_length`` rows will be used to generate a sliding window.
        :param window_max_count: Optional. Number of rows that should be used to generate input examples. If the 
            total row count is larger than this number, the input data will be randomly sampled to hit the count.
```
Small nit on [formatting the docstring params](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#an-example-class-with-docstrings) in the Sphinx RTD format for the [Python API docs](https://airflow.apache.org/docs/apache-airflow-providers-google/stable/_api/airflow/providers/google/cloud/hooks/vertex_ai/auto_ml/index.html#airflow.providers.google.cloud.hooks.vertex_ai.auto_ml.AutoMLHook.create_auto_ml_forecasting_training_job).